### PR TITLE
documentation harmonization part 1

### PR DESCRIPTION
**Pull request by zramsay**
- re-organizes the documentation to allow for clean sequence on website.
- closes https://github.com/eris-ltd/eris/issues/1163
- docs need to be tested both manually & by setting up simple tests

### CODE REVIEW

File: docs/chain-deploying.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+```bash
+eris chains start multichain --init-dir ~/.eris/chains/multichain_validator_002 --logrotate
+```
+
+And voila! You multi-node, permissioned chain is started!
+
+## Step 5: Start some services
+
+We're now going to start a few services which help us manage cloud instances. You'll notice we used the `--logrotate` flag when starting the chains. This service is **absolutely essential** when working with cloud boxes. We have had **dozens** of cloud nodes overfill on us due to logs overloading the allocated storage space on the node. To overcome this, we use a [logs rotator service](https://github.com/tutumcloud/logrotate) which discards the old logs. If you forgot to use the flag, don't fret! `eris services start logrotate` will get you squared away.
```
Comment (VoR0220) [inline docs/chain-deploying.md]: ++

File: docs/chain-deploying.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+
+[^1]
+
+Understanding the ports is important for distributed software. If the blockchains *think* they are running on port X, but that port is exposed to the internet as port Y when they are doing their advertising to their peers they will be saying, "Hey, I'm located on IP address Z on port X". But the problem is that from the internet's perspective they should really be saying "Hey, I'm located on IP address Z on port Y".
+
+So at Eris we routinely recommend that you simply "flow through" the ports rather than trying to do anything funky here; this means that whatever port you select in the `laddr` fields and in the chain definition file, that you publish the same port on the host (meaning don't have something like `11111:46656` in your chain definition file). It can be made to work, but it requires some doing to do that right. But for now we will only be running one chain on each of our cloud validators so there will not be any port conflicts to worry about.
```
Comment (VoR0220) [inline docs/chain-deploying.md]: at Eris? Or at Monax?

Comment (zramsay) [inline docs/chain-deploying.md]: good catch

Comment (zramsay) [inline docs/chain-deploying.md]: although let's deal with this rabbit hole on 0.17

Comment (VoR0220) [inline docs/chain-deploying.md]: kk

**Pull request merged**